backend/internal/usecase/post_usecase_test.go

```
@@ -0,0 +1,708 @@
+}
+
+// fakeFollowerNotifier は FollowerNotifier の fake。
+// フォロワー通知は goroutine で実行されるため、完了を待てるように WaitGroup を持つ。
+type fakeFollowerNotifier struct {
+	mu            sync.Mutex
+	followerIDs   []uint
+	findErr       error
+	createErr     error
+	notifications []*model.Notification
+	called        int
+	wg            sync.WaitGroup
```

golangci-lint (2.12.2)

[error] field wg is unused

(unused)